docs/flocker-features/gce-configuration.rst

```diff
@@ -0,0 +1,93 @@
+.. _gce-dataset-backend:
+
+=====================================
+Google GCE / PD Block Device Backend 
```

Comment:
+1

Sorry about that. I'm overly familiar with PD being persistent disk from my previous job.